packages/react-reconciler/src/ReactFiberLane.new.js

```diff
@@ -459,6 +459,9 @@ export function includesOnlyNonUrgentLanes(lanes: Lanes) {
   const UrgentLanes = SyncLane | InputContinuousLane | DefaultLane;
   return (lanes & UrgentLanes) === NoLanes;
 }
+export function includesOnlyTransitions(lanes: Lanes) {
```

Comment:
I left the new `includesOnlyNonUrgentLanes` in the codebase because it's used by the deferred value logic too now.